### PR TITLE
New Extention: Crunchbase

### PR DESCRIPTION
**Pull request by qdequele**
## Description

Add a new extension, `crunchbase` that allows searching on the Crunchbase websites thanks to the Meilisearch search engine. 

## Screencast
Crunchbase Raycast Screenshot

https://user-images.githubusercontent.com/6064892/195601833-7dc060f7-5fbc-4b75-89f2-756e11bafde8.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/crunchbase/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                  ) : null}
+                  {searchResult.twitter_url ? (
+                    <Action.OpenInBrowser title="Open Twitter" url={searchResult.twitter_url} />
+                  ) : null}
+                  {searchResult.linkedin_url ? (
+                    <Action.OpenInBrowser title="Open Linkedin" url={searchResult.linkedin_url} />
+                  ) : null}
+                </ActionPanel.Section>
+                <ActionPanel.Section>
+                  <Action title="Toggle Detail" onAction={() => setShowingDetail(!showingDetail)} />
```
Comment (pernielsentikaer) [inline extensions/crunchbase/src/index.tsx:83]: Missing an icon

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Seems like some of the images missing a source, could you look into this?

CleanShot 2022-10-14 at 11 05 43@2x

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/crunchbase/src/index.tsx (hunk 1 above)
Comment (qdequele) [reply extensions/crunchbase/src/index.tsx]: Thanks, I've fixed this 👍

**Comment by qdequele** [comment]
Hi @pernielsentikaer, thanks for the review. I've fixed the metadata image and the panel icon that was missing.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

One thing I forgot to mention yesterday, do you think it's possible to add a throttle?

That would indeed make the UX even better 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by qdequele** [comment]
Hi, I just added the throttle on search. Thanks again for your review.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**